test/test_endpoints.py

```diff
@@ -350,3 +381,147 @@ def test_get_transaction_trace_of_rejected():
     resp_body = resp.json()
     assert resp_body["code"] == str(StarknetErrorCode.NO_TRACE)
     assert resp.status_code == 500
+
+
+@devnet_in_background()
+def test_send_message_to_l2_deploy_execute():
+    """Test POST l1 to l2 deploy contract and execute transaction"""
+    # Deploy L1L2 contract
+    deploy_info = deploy(contract=L1L2_CONTRACT_PATH)
+
+    # Create l1 to l2 mock transaction
+    response = send_message_to_l2(
+        {
+            "l2_contract_address": deploy_info["address"],
+            "entry_point_selector": "0xC73F681176FC7B3F9693986FD7B14581E8D540519E27400E88B8713932BE01",
```

Comment:
This entry_point_selector should be extracted and it should be documented what it is. I would recommend using `starknet.public.abi.get_selector_from_name` on the actual name:
```python
DEPOSIT_SELECTOR = get_selector_from_name("deposit")
```

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/365/commits/53c4ee8b0993637d98d886a8e28ca7884dc4cf65